### PR TITLE
chore: added adr-65 and adr-66 about avatar system and emotes

### PR DESCRIPTION
**Pull request by AjimenezDCL**

### CODE REVIEW

File: docs/ADR-65-avatar-system-for-renderer.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+## Benefits
+
+- Modularized: Every responsibility of the loading process is a separated module completly decoupled from the rest. It directly impact in the time required to find and fix bugs, maintain and scale each module.
+- Mockable: Test suite is more reliable, every aspect of the process can be tested without requiring visual tests.
+- Adaptability: Implementing new ways of loading and rendering avatars it’s easier. i.e. a LOD system with more than 3 levels can be implemented in Desktop without altering any other module.
+
+## Competition
+
+No alternatives have been consider.
```
Comment (menduz) [inline docs/ADR-65-avatar-system-for-renderer.md:368]: Danger

File: docs/ADR-65-avatar-system-for-renderer.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
+# ADR 65 Avatar System for Renderer (Unity)
+
+## Problem Statement
+
+Avatars are a key part of Decentraland, the system behind the curtains must be resilient, scalable and performant.
```
Comment (menduz) [inline docs/ADR-65-avatar-system-for-renderer.md:5]: Could you add more details about what people is about to read? Does this document solves an issue? a design problem? a refactor? Although I understand that avatars are key part of DCL, it is not clear what decision is being addressed in this document

**Review by menduz — CHANGES_REQUESTED** [review]

File: docs/ADR-65-avatar-system-for-renderer.md (hunk 1 above)
Comment (menduz) [reply docs/ADR-65-avatar-system-for-renderer.md:368]: the sentence `No alternatives have been consider.` is misleading and adds no context to the decision. 

I'd prefer you to add `The competition is the current system, which flaws are X, Y and Z. Adding to that, we are normalizing the implementation following best practices in the industry ...` or completely remove it.

Comment (AjimenezDCL) [reply docs/ADR-65-avatar-system-for-renderer.md:368]: Fair enough! changing it in both ADRs

**Review by menduz — APPROVED** [review]

**Pull request merged**